air/src/options.rs

```diff
@@ -18,20 +20,52 @@ impl ProvingOptions {
     // --------------------------------------------------------------------------------------------
 
     /// Standard proof parameters for 96-bit conjectured security in non-recursive context.
-    pub const REGULAR_96_BITS: WinterProofOptions =
-        WinterProofOptions::new(27, 8, 16, FieldExtension::Quadratic, 8, 255);
+    pub const REGULAR_96_BITS: WinterProofOptions = WinterProofOptions::new(
+        27,
+        8,
+        16,
+        FieldExtension::Quadratic,
+        8,
+        255,
+        BatchingMethod::Algebraic,
+        BatchingMethod::Algebraic,
+    );
```

Comment:
Q: Since all the parameters are the same except for switching from linear to algebraic batching, shouldn't we expect a decrease in security (i.e. this wouldn't be 96 bits)?

`Proof::security_level()` does return `96`, but I'm not sure why

Reply:
For conjectured security, the type of batching should not affect the security bits in the current implementation.